vxtelegram/telegram.py

```diff
@@ -264,24 +268,31 @@ def handle_inbound_inline_query(self, message_id, inline_query):
 
     def translate_inbound_message(self, message):
         """
-        Translates inbound Telegram message into Vumi's default format.
+        Translates inbound Telegram message into Vumi's default format. We want
+        to use the user's username as from_addr if possible, for readability.
         """
-        telegram_id = message['message_id']
+        telegram_msg_id = message['message_id']
         content = message['text']
         to_addr = self.bot_username
 
         # Messages sent over channels do not contain a 'from' field - in that
         # case, we want the channel's chat id
         if 'from' in message:
-            from_addr = message['from']['id']
+            from_addr = message['from']['username']
+            from_addr_type = 'telegram_username'
```

Comment:
I would make `telegram_username` into a module-level constant so people can use it, e.g.
```python
AT_TELEGRAM_USERNAME = 'telegram_username'
```